packages/migration/src/tests/mods/7.0.0/__snapshots__/navPackageMove.test.ts.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Nav package move moves import statements 1`] = `
+"import {  } from 'office-ui-fabric-react'
```

Comment:
Hmmm would have expected this line not to be present...

Reply:
Agreed. Fixed.